pkg/publicapi/client/v2/client.go

```diff
@@ -73,18 +73,13 @@ func (c *httpClient) Get(ctx context.Context, endpoint string, in apimodels.GetR
 		return apimodels.NewUnauthorizedError("invalid token")
 	}
 
-	var apiError *apimodels.APIError
 	if resp.StatusCode != http.StatusOK {
-		apiError, err = apimodels.FromHttpResponse(resp)
-		if err != nil {
-			return err
+		apiError := apimodels.GenerateAPIErrorFromHTTPResponse(resp)
+		if apiError != nil {
+			return apiError
```

Comment:
nit: compact this and others:
```suggestion
		if apiError := apimodels.GenerateAPIErrorFromHTTPResponse(resp); apiError != nil {
		    // code and stuff
		}
```

Reply:
Thanks for the suggestion. Code changed.